src/components/App/App.tsx

```diff
@@ -1,44 +1,30 @@
-import React, { useState, useEffect } from 'react';
+import React, {useState, useContext, useEffect} from 'react';
+import {QuestionContext, AllQuestionFormat} from '../../types'
+import {getQuestions} from '../../apiCalls'
 
-//types
-import { data } from '../../types';
 //components
-// import Home from '../Home/Home';
-// import Error from '../Error/Error';
-// import Journey from '../Journey/Journey';
-import Sourvey from '../Sourvey/Sourvey';
-import NavBar from '../NavBar/NavBar';
-// import GenerateReport from '../GenerateReport/GenerateReport';
+import Home from '../Home/Home';
+//import Error from '../Error/Error';
+//import Journey from '../Journey/Journey';
+import {Sourvey} from '../Sourvey/Sourvey';
+//import GenerateReport from '../GenerateReport/GenerateReport';
+//import './App.css';
 
-import './App.css';
+//create interface for context
 
-const Context = React.createContext({})
-
-
-const App = () => {
-
-  const [data, setData] = useState<data>();
-
-  const getQuestionsData = () => {
-
-    // setData();
-  }
+const App:React.FC = () =>{
+  const [questions, updateQuestions] = useState<AllQuestionFormat | null>(null);
+  useEffect(() => {
+    getQuestions().then((data) => updateQuestions(data))
+  }, []);
 
   return (
-    <>
-    {/* //create interface for data
-    // <Context.Provider data={data}>
-    //   <Home />
-    //   <Error />
-    //   <Journey /> */}
-    
-    <NavBar />
-    <Sourvey />
-    {/* //  <GenerateReport />
-    // </Context.Provider> 
-    */}
-    </>
-    );
-}
+    <QuestionContext.Provider value={questions}>
+      <div>
+        <Home />
+        <Sourvey />
+      </div>
+     </QuestionContext.Provider>
```

Comment:
Just to make sure, as long as the component is wrapped within the provider  "tag" they will have access to the state, correct?

Reply:
Yep! And the `value={}` of the provider is essentially where the information lives. In this case the value is state, so if that state is updated on App, all children will have access to the updated info and re-render if the diffing algo determine neccessary.

Reply:
Only App will be wrapped in the provider tag, which gives every component rendered within that wrapper access to the context state via accessing useContext.